src/coprocessor_v2/plugin_registry.rs

```diff
@@ -78,7 +250,15 @@ impl LoadedPlugin {
         // Leak library so that we will never drop it
         std::mem::forget(lib);
```

Comment:
Current approach looks good to me, that we never drop loaded libraries. I have an idea about this: if there is no reference to `LoadedPlugin`, can we ensure that no code of plugin is running? If so, we could safely drop this lib upon last instance of `Arc<LoadedPlugin>` is dropped.

Reply:
> if there is no reference to LoadedPlugin, can we ensure that no code of plugin is running

No, we can't. `'static` reference from the library will be leaked into the runtime even no borrow rule is violated.